src/uk/gov/hmcts/contino/GithubAPI.groovy

```diff
@@ -49,16 +52,30 @@ class GithubAPI {
       requestBody: "${body}",
       consoleLogResponseBody: true,
       validResponseCodes: '200')
+
+    this.cachedLabelList = []
```

Comment:
needs testing, not 100% sure `this` will work in a static context, may work though, https://github.com/hmcts/cnp-jenkins-library/blob/master/src/uk/gov/hmcts/pipeline/TeamConfig.groovy#L41 refers to it with no qualifier

Reply:
Looking at this further, I'm not at all certain that cachedLabelList should be static.  My understanding is that a static property exists once, rather than for each instance of a class - which would absolutely break this given that each instance will be returning it's own list of labels (from different Repo's) to store in the cache.